scripts/scripts.js

```diff
@@ -231,9 +264,29 @@ async function loadPage() {
     if (lastSection) lastSection.insertAdjacentElement('beforeend', a);
   }
 
-  setConfig({ ...CONFIG, miloLibs: LIBS });
+  const eventConfigItems = {
+    decorateArea: () => decorateEvent(document),
+    externalLibs: [
+      {
+        base: EVENT_LIBS,
+        blocks: EVENT_BLOCKS, // or your custom EVENT_BLOCKS_OVERRIDE
+      },
+    ],
+  };
+
+  setConfig({ ...CONFIG, ...eventConfigItems, miloLibs: LIBS });
+  setEventConfig({ cmsType: 'DA' }, CONFIG);
+
+  if (isEventMetadata) decorateEvent(document);
+
   loadLana({ clientId: 'bacom', tags: 'info', endpoint: 'https://business.adobe.com/lana/ll', endpointStage: 'https://business.stage.adobe.com/lana/ll' });
   transformExlLinks(getLocale(CONFIG.locales));
+
+  if (isEventMetadata) {
+    const { eventsDelayedActions } = await import(`${EVENT_LIBS}/libs.js`);
```

Comment:
Do we need to await if we are delayed and before load area?

Reply:
If we do not await, won't there be a chance that `eventsDelayedActions()` will be called before the lib is imported?

Reply:
I made some changes, please take a look when you get a second!